app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -847,9 +847,9 @@ def build_toolbar_hide_button(id)
       when "vm_reconfigure"
         return true unless @record.reconfigurable?
       when "vm_retire"
-        return true unless @record.is_available?(:retire)
+        return true unless @record.supports_retire?
       when "vm_retire_now"
-        return true unless @record.is_available?(:retire_now)
+        return true unless @record.supports_retire?
```

Comment:
@durandom so no code change in PR to account for the difference in functionality?